afk-daily.sh

```diff
@@ -975,8 +1007,8 @@ function legendsTournament() {
     else
         input tap 550 1450
     fi
-    sleep 2
-    input tap 550 280
+    sleep 2.5
+    doSkip
```

Comment:
This is at a completely wrong place. First, you removed the collection of the Chest and secondly you wanted to skip a battle while not even being inside one yet. I'm guessing this was a typo that you didn't test in the end...?
Either way, this will be fixed in the soon-to-be-commited changes. :)

Reply:
Oops, I think my eyes were crossing each other :sweat_smile:  

And no, didn't have any remaining battle that day :wink: